packages/cloud-cognitive/src/components/NonLinearReading/NonLinearReading.js

```diff
@@ -59,19 +59,17 @@ export let NonLinearReading = React.forwardRef(
         <button
           type="button"
           aria-expanded={isOpen}
-          className={cx(`${blockClass}__term-${theme}`, [
+          className={cx(`${blockClass}__keyword`, [
             isOpen
-              ? [`${blockClass}__term-${theme}--open`]
-              : [`${blockClass}__term-${theme}--closed`],
+              ? [`${blockClass}__keyword-open`]
+              : [`${blockClass}__keyword-closed`],
           ])}
           onClick={handleToggle}
         >
           {children}
-          {isOpen && <ChevronUp16 />}
+          <ChevronDown16 />
```

Comment:
The chevron is always visible now, only rotated 180 degrees depending on the open/closed state of the keyword button.